_episodes/20-parallel-raster-computations.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "Parallel raster computations using Dask"
+teaching: 40
+exercises: 20
+questions:
+- "How can I parallelize computations on rasters with Dask?"
```

Comment:
```suggestion
- "How can I run several computations on rasters simultaneously (parallelization) with Dask?"
```

Reply:
Uhm, this is actually slightly different (we show how to parallelize a single raster calculation, not how to run multiple calculations in parallel). What about:

"How can I exploit parallelization in my raster calculations?"

Reply:
yes, much better, thanks.